### ISSUE TITLE
Cannot change playback speed when using VoiceOver

### ISSUE THREAD

**Issue by MarcoZehe** [signalapp/Signal-iOS#6037]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
When playing back voice messages, and having VoiceOver running, it is not possible to change the playback speed. VoiceOver doesn't see the button to switch playback speed, and there is no custom action on the message item to accomplish the same task'

### Steps to reproduce
- Turn on 
- VoiceOverOpen a chat that contains a voice message and play 
- Try to find the button to toggle playback speed by either exploring by touch or flicking left and right through elements.
- Flick up and down through the custom rotor actions to find an action to change playback speed.

**result:** Neither touch exploration nor flicking through elements will find the button. Flicking up and down through custom rotor actions also doesn't give you an action to change playback speed.

**Expected result:** VoiceOver should either find the button or have a custom action available to toggle playback speed. A custom action is the preferred fix for this.

### Device info

**Device**: iPhone 16 Pro Max

**iOS version**: 18.4.1

**Signal version:** 7.55

**Comment by 2-of-1** [comment]
Also jumping to a quoted message doesn't work with VoiceOver.

**Comment by DO9RE** [comment]
> 
> 
> - [x] I have searched open and closed issues for duplicates
> - [x] I am submitting a bug report for existing functionality that does not work as intended
> - [x] This isn't a feature request or a discussion topic
> 
> ----------------------------------------
> 
> ### Bug description
> When playing back voice messages, and having VoiceOver running, it is not possible to change the playback speed. VoiceOver doesn't see the button to switch playback speed, and there is no custom action on the message item to accomplish the same task'
> 
> 
> ### Steps to reproduce
> - Turn on 
> - VoiceOverOpen a chat that contains a voice message and play 
> - Try to find the button to toggle playback speed by either exploring by touch or flicking left and right through elements.
> - Flick up and down through the custom rotor actions to find an action to change playback speed.
> 
> **result:** Neither touch exploration nor flicking through elements will find the button. Flicking up and down through custom rotor actions also doesn't give you an action to change playback speed.
> 
> **Expected result:** VoiceOver should either find the button or have a custom action available to toggle playback speed. A custom action is the preferred fix for this.
> 
> ### Screenshots
> 
> 
> 
> ### Device info
> 
> **Device**: iPhone 16 Pro Max
> 
> **iOS version**: 18.4.1
> 
> **Signal version:** 7.55
> 
> ### Link to debug log
> 
> 
> 

I can copy on that.

**Comment by MarcoZehe** [comment]
This issue is still a problem in the current app store version of Signal for iOS.

**Comment by sashaweiss-signal** [comment]
Thanks for confirming, @MarcoZehe. We're tracking this internally as well.
